Detectors/FIT/FT0/calibration/CMakeLists.txt

```diff
@@ -85,3 +85,4 @@ o2_add_library(FT0Calibration
         PUBLIC_LINK_LIBRARIES
         O2::DataFormatsFT0 O2::FITCalibration
         )
+o2_data_file(COPY files DESTINATION Detectors/FT0/)
```

Comment:
I think you're missing the newline at the end of file. But since all tests passed, I think this is not a problem